README.md

```diff
@@ -1,2 +1,37 @@
 # js-assertion-cheat-sheet
+
 This repository contains examples of various, often used, and tricky assertions in the common JS assertion tools.
+
+## CI/CD
+
+### Feature Branch deployment
+
+> [!IMPORTANT]  
+> Before deploying any feature branches, make sure that the AWS CDK Stack is already deployed.
+
+#### Deploying a Feature Branch
+
+To deploy a feature branch, manually execute the [Feature Branch Deployment](https://github.com/Idea-Pool/js-assertion-cheat-sheet/actions/workflows/feature-branch-deployment.yaml) workflow.
+
+> [!NOTE]  
+> Make sure to select your feature branch from the dropdown before executing the workflow.
+
+##### Redeployment
+
+Feature branches can be redeployed by re-executing the deployment workflow.
+
+#### Cleaning up a Feature Branch
+
+Once a feature branch is merged or no longer needed, clean it up by executing the [Feature Branch Cleanup](https://github.com/Idea-Pool/js-assertion-cheat-sheet/actions/workflows/feature-branch-cleanup.yaml) workflow.
+
+### AWS CDK Stack deployment
+
+The AWS CDK stack sets up the necessary infrastructure for the application. It needs to be deployed only once initially, and then it remains active until explicitly destroyed.
+
+#### Deploying the stack
+
+Deploy the stack by manually executing the [Stack Deployment](https://github.com/Idea-Pool/js-assertion-cheat-sheet/actions/workflows/stack-deployment.yaml) workflow.
```

Comment:
Can you check if we can limit the permissions who can execute these workflows to a given team?

Reply:
We could use [Manual Workflow Approval](https://github.com/marketplace/actions/manual-workflow-approval) action so that before every stack deployment/destroy, someone from a given team would have to approve the workflow execution.